.github/workflows/main.yml

```
@@ -82,3 +111,68 @@
           name: published-app
           path: backend/Api/Server/publish/
+
+  deploy:
+    name: Deploy to Server
+    runs-on: ubuntu-latest
+    needs: publish
+    timeout-minutes: 45  # ⏱️ Extended deployment timeout
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)